### PR TITLE
Rename ArtNet, Artnet, artnet, Art-net to Art-Net

### PR DESCRIPTION
**Pull request by nils-van-zuijlen**
In all comments, strings and documentation.

#1328

### CODE REVIEW

**Comment by peternewman** [comment]
Amazing thanks @nils-van-zuijlen

File: plugins/artnet/ArtNetNode.cpp [hunk 1]
```diff
@@ -1334,11 +1334,13 @@ void ArtNetNodeImpl::RDMRequestCompletion(
       // hopefully update the remote controller
       port->on_discover->Run();
     } else {
-      OLA_WARN << "ArtNet RDM request failed with code " << reply->StatusCode();
+      OLA_WARN << "Art-Net RDM request failed with code "
+               << reply->StatusCode();
```
Comment (peternewman) [inline plugins/artnet/ArtNetNode.cpp]: You've somehow inserted a Unicode character in here according to the compiler errors.

Comment (nils-van-zuijlen) [reply plugins/artnet/ArtNetNode.cpp]: Yes, it was an non-breaking space instead of a regular space.

**Comment by nils-van-zuijlen** [comment]
The travis build failed because of [this commit in codespell](https://github.com/lucasdemarchi/codespell/commit/c30e3fb456682e170e9f788431e59ad10e86223b)

File: NEWS [hunk 2]
```diff
@@ -653,19 +653,19 @@ Documentation:
  * Added a configure test for the python google.protobuf module
  * Added the ola_trigger program
  * Changed the ForceDiscovery rpc to only return once discovery completes
- * Fixed ArtNet RDM discovery so DMXWorkshop works correctly - #160
- * Fixed a memory leak in the ArtNet RDM implementation
- * Add an option for using the limited broadcast address for ArtNet dmx data
+ * Fixed Art-Net RDM discovery so DMXWorkshop works correctly - #160
+ * Fixed a memory leak in the Art-Net RDM implementation
+ * Add an option for using the limited broadcast address for Art-Net dmx data
```
Comment (peternewman) [inline NEWS]: Nit, and not your fault, but could we change dmx to DMX here please.

File: data/rdm/manufacturer_pids.proto [hunk 3]
```diff
@@ -2103,7 +2103,7 @@ manufacturer {
         }
         label {
           value: 1
-          label: "ArtNet"
+          label: "Art-Net"
```
Comment (peternewman) [inline data/rdm/manufacturer_pids.proto]: These also want updating here, which is where this data is sourced from if you're feeling keen: https://github.com/OpenLightingProject/rdm-app/blob/master/data/pid_data.py#L719

File: plugins/artnet/ArtNetNode.cpp [hunk 4]
```diff
@@ -1334,11 +1334,13 @@ void ArtNetNodeImpl::RDMRequestCompletion(
...
     } else {
-      OLA_WARN << "ArtNet RDM request failed with code " << reply->StatusCode();
+      OLA_WARN << "Art-Net RDM request failed with code "
+               << reply->StatusCode();
     }
   } else {
     // the universe address has changed we need to drop this request
-    OLA_WARN << "ArtNet Output port has changed mid request, dropping response";
+    OLA_WARN << "Art-Net Output port has changed mid request, "
+             <<"dropping response";
```
Comment (peternewman) [inline plugins/artnet/ArtNetNode.cpp]: Can we have a space after << please. Also we'd generally just move the one word that doesn't fit down (i.e. response in this case).

File: plugins/e131/messages/E131ConfigMessages.proto [hunk 5]
```diff
@@ -14,7 +14,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  *
  * E131ConfigMessages.proto
- * Defines the protocol buffers used to configure the Artnet device
```
Comment (peternewman) [inline plugins/e131/messages/E131ConfigMessages.proto]: @nils-van-zuijlen Sorry I've just spotted, do you fancy opening a new PR to correct this to E1.31 as it's actually a copy/paste error from then?

Comment (nils-van-zuijlen) [reply plugins/e131/messages/E131ConfigMessages.proto]: But isn't it the filename ?

Comment (peternewman) [reply plugins/e131/messages/E131ConfigMessages.proto]: Yes, the first one should be. I meant line 17 where it talks about using E131ConfigMessages.proto to configure the 'Artnet' device!

**Pull request merged**